libs/service-portal/settings/personal-information/src/components/Forms/ProfileForm/components/Inputs/Email.tsx

```diff
@@ -219,7 +221,7 @@ export const InputEmail: FC<Props> = ({
                 checkSetPristineInput()
               }}
               placeholder="nafn@island.is"
-              error={errors.email?.message || formErrors.email}
+              error={(errors.email?.message as string) || formErrors.email}
```

Comment:
Curious why this is necessary :(

Reply:
We could also just check for the type of the message or make the controller accept the new types that are possible in message but I am afraid that I might break some older logic. Most of the type casting is just to avoid breaking older stuff.

Reply:
Addressed, everything was easier with the newest typescript changes.